### PR TITLE
made code quality improvements

### PR DESCRIPTION
**Pull request by KelseyStiff**
I've fixed spelling errors, removed unused codes/redundant comments, removed and added spaces where needed, changed variable names.

### CODE REVIEW

**Comment by claraj** [comment]
@svenJonis138 can you review?

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,65 @@
...
-
-
-import json  # import json for json processing
 def main():
+    """
+    Read data from JSON file to start main method
+    """
     with open('data.json','r') as file:
         data = file.read()
         data = json.loads(data)
```
Comment (claraj) [inline convert_state.py]: Use two different variables. Re-using data for a string read from a file, and then using it again for the dictionary is not good style. "data" is also a bad variable name - be more specific

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,65 @@
...
-    print(data)
     
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    state_abbreviations = data #dictionary of state abbreviations keys and state name values
```
Comment (claraj) [inline convert_state.py]: you can avoid this assignment by fixing the data variable name issue above

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,65 @@
...
+def convert_state_to_abbreviation(dictionary):
+    """
+    Converts user input for full state name into state abbrevation
+    """
+    user_input = input('Enter state name: ').capitalize()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('state not found.')
+    else:
+        print ( 'The abbreviation for ' +  user_input + 'is' + result)
```
Comment (claraj) [inline convert_state.py]: Recommend f-string instead of concatenation

File: convert_state.py [hunk 4]
```diff
@@ -1,60 +1,65 @@
...
+    """
+    Converts user input for abbrevation into full state name 
+    """
+    user_input = input('Enter abbreviation name: ').upper()
+    result = dictionary.get(user_input)
+
+    if result is None:
+        print('abbreviation not found.')
+    else:
+        print('The state with the abbreviation ' + user_input + 'is' + result)
```
Comment (claraj) [inline convert_state.py]: Recommend f-string instead of concatenation

**Review by claraj — CHANGES_REQUESTED** [review]
Looks much better! Just a few minor things

File: convert_state.py (hunk 1 above)
Comment (svenJonis138) [reply convert_state.py]: looks like Clara beat me to this @KelseyStiff

Comment (KelseyStiff) [reply convert_state.py]: @claraj  @svenJonis138 thanks for the feedback! I changed the variable names to be different and more descriptive.

File: convert_state.py (hunk 2 above)
Comment (KelseyStiff) [reply convert_state.py]: @claraj fixed sounds good, I resolved the above issue and removed this variable assignment.

File: convert_state.py (hunk 4 above)
Comment (KelseyStiff) [reply convert_state.py]: @claraj f-string added

File: convert_state.py (hunk 3 above)
Comment (KelseyStiff) [reply convert_state.py]: @claraj f-string added

**Comment by KelseyStiff** [comment]
@claraj Thanks for the feedback, I believe all outstanding issues are now resolved.

**Review by claraj — APPROVED** [review]

**Pull request closed without merging**